code/game/objects/effects/spawners/decorative_spawners.dm

```diff
@@ -113,6 +113,14 @@
 	/datum/nothing = 7,
 	/obj/effect/decal/cleanable/dirt = 1)
 
+/obj/effect/spawner/random_spawners/fungus_frequent
+	name = "fungus frequent"
+	icon_state = "fungus"
+	color = "#D5820B"
+	result = list(
+		/datum/nothing = 1,
+		/obj/effect/decal/cleanable/fungus = 1)
+
```

Comment:
Could you use the new spawner type for this? Something like:

```suggestion
/obj/effect/spawner/random/fungus
	icon_state = "fungus"
	color = "#D5820B"
	loot = list(/obj/effect/decal/cleanable/fungus)

/obj/effect/spawner/random/fungus/frequent
	spawn_loot_chance = 50
```

I know the rest of these are using `/obj/effect/spawner/random_spawners` but I'm working to phase them out, `/obj/effect/spawner/random` has a lot more features and doesn't require constantly adding a `/datum/nothing`.

Reply:
Sure, sounds good